JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -122,11 +120,15 @@ fun HomeScreen(
  * Stateless composable is not coupled to any specific state management.
  *
  * @param posts (state) the data to show on the screen
+ * @param favorites (state) favorite posts
+ * @param onToggleFavorite (event) toggles favorite for a post
  * @param onRefreshPosts (event) request a refresh of posts
- * @param onErrorDismiss (event) request the current error be dismissed
+ * @param onErrorDismiss (event) indicate an error can be dismissed
```

Comment:
Why the change? Isn't "request" correct here as the caller can ignore the event to veto it.